Misc/NEWS.d/next/Library/2024-11-05-11-28-45.gh-issue-126451.XJMtqz.rst

```diff
@@ -0,0 +1,2 @@
+Register the :class:`!contextvars.Context` type to
+:class:`!collections.abc.Mapping`.
```

Comment:
```suggestion
Register the :class:`contextvars.Context` type to
:class:`collections.abc.Mapping`.
```

Both are present, should work without `!` 🤔

Reply:
Thanks, I'll admit that I copied that syntax from another NEWS entry without bothering to look up what it meant exactly...